packages/iobridge-commons/src/message/cross-chain-message.ts

```diff
@@ -0,0 +1,56 @@
+import { recoverPersonalSignature } from '@metamask/eth-sig-util';
+import { helpers } from '@ckb-lumos/lumos';
+
+interface EthereumRpc {
+  (payload: { method: 'personal_sign'; params: [string /* from */, string /* message */] }): Promise<string>;
+}
+
+export interface EthereumProvider {
+  selectedAddress: string;
```

Comment:
it is not necessary since `request('personal_sign', [msg, account])` handled account